classes/template/TemplateManager.inc.php

```diff
@@ -79,13 +68,13 @@ public function initialize($request) {
 				// Assign page header
 				$this->assign('displayPageHeaderTitle', $context->getPageHeaderTitle());
 				$this->assign('displayPageHeaderLogo', $context->getPageHeaderLogo());
-				$this->assign('numPageLinks', $context->getSetting('numPageLinks'));
-				$this->assign('itemsPerPage', $context->getSetting('itemsPerPage'));
-				$this->assign('enableAnnouncements', $context->getSetting('enableAnnouncements'));
-				$this->assign('disableUserReg', $context->getSetting('disableUserReg'));
+				$this->assign('numPageLinks', $context->getData('numPageLinks'));
```

Comment:
More syntactical pedantry :) I've been gradually moving series of calls like this over to `$templateMgr->assign(array(...));`